openvpn/process.go

```diff
@@ -108,5 +122,11 @@ func (openvpn *openvpnProcess) Stop() {
 		openvpn.management.Stop()
 	}()
 
+	waiter.Add(1)
+	go func() {
+		defer waiter.Done()
+		openvpn.tunnelSetup.Teardown()
```

Comment:
This tunnel tear down must be called ONLY AFTER openvnp process is completed, to avoid funny openvpn or tunnel setup fail errors. It cannot be go routined.